Kajabi/sage-lib
# Loader - improve loading indicator

## Description

made style improvements to the bar and spinner loader
- [x] - updated loader animation
- [x] - updated Rails component
- [x] - updated React component
- [x] - updated a11y for both variations of component

### Screenshots


|  before  |  after  |
|--------|--------|
|![loaders-before](https://user-images.githubusercontent.com/1241836/101224422-27558180-3654-11eb-9c9c-25f562cbcb14.gif)|![loaders-after](https://user-images.githubusercontent.com/1241836/101224435-2de3f900-3654-11eb-961f-91abd806ca3d.gif)|


## Test notes


### Steps for testing
1a. Visit the loader page Rails page and verify: http://localhost:4000/pages/element/loader
1b. Visit the loader page React page and verify: http://localhost:6006/?path=/story/sage-loader--default
